app/v2/notifications/post_notifications.py

```diff
@@ -218,15 +229,15 @@ def process_sms_or_email_notification(*, form, notification_type, api_key, templ
         notification_type=notification_type,
         api_key_id=api_key.id,
         key_type=api_key.key_type,
-        client_reference=form.get('reference', None),
+        client_reference=form.get("reference"),
```

Comment:
Why remove the default value of None?

Reply:
It's superfluous.  If you don't specify a default, None is the default.